From: "Steven Rostedt (Red Hat)" <rostedt@goodmis.org>
Date: Wed, 1 May 2013 09:38:20 -0400
Subject: [PATCH 303/307] rcutiny: Fix typo of using swake_up() instead of
 swait_wake()
Origin: https://git.kernel.org/cgit/linux/kernel/git/rt/linux-stable-rt.git/commit?id=e07b0011832bfe0df4565bdb4e24b8e5e5cfc31d

Converting the rcutiny wake up queues to use the simple wait queues
there was a typo in one of the functions.

Reported-by: Dmitry Lysenko <dvl36.ripe.nick@gmail.com>
Signed-off-by: Steven Rostedt <rostedt@goodmis.org>
---
 kernel/rcutiny_plugin.h | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/kernel/rcutiny_plugin.h b/kernel/rcutiny_plugin.h
index f0a6606..6242ccd 100644
--- a/kernel/rcutiny_plugin.h
+++ b/kernel/rcutiny_plugin.h
@@ -883,7 +883,7 @@ static void rcu_preempt_process_callbacks(void)
 static void invoke_rcu_callbacks(void)
 {
 	have_rcu_kthread_work = 1;
-	swake_up(&rcu_kthread_wq);
+	swait_wake(&rcu_kthread_wq);
 }
 
 /*
